src/Book/Book.tsx

```diff
@@ -107,13 +107,10 @@ const Book = (props: Props) => {
         promise.then(res => {
             notifySuccess(res.expirationDate);
         })
-            .catch(e => {
-                if(e.status === BAD_REQUEST) notifyError('No se pudo realizar el préstamo ya que tiene prestamos atrasados.');
-                else if (e.status === TOO_MANY_REQUESTS) notifyError('No se pudo realizar el préstamo ya que tiene demasiados prestamos activos.');
-                else if (e.status === NOT_ACCEPTABLE) notifyError('No se pudo realizar el préstamo ya que ya tiene un prestamo de este libro.');
-                else if (e.status === EXPECTATION_FAILED) notifyError('Este libro no tiene ejemplates disponibles.');
-                else if (e.status === UNAUTHORIZED) notifyError('Debe ser un alumno para realizar un prestamo.');
-                else notifyError('Error inesperado. Intente de nuevo.');
+            .catch((error) => {
+                error.then((e:any)=>{
+                    notifyError(e);
+                });
```

Comment:
I make the comment here but the same applies to the other files.
Is preferable to throw the error message or object instead of the promise (in the http.tsx), this way we avoid making this `error.then()`